BedrockServer.h

```diff
@@ -3,7 +3,6 @@
 #include <libstuff/libstuff.h>
```

Comment:
I'll get this in the next PR, I'm doing a bunch of work in this file for multi-threading anyway.